client/src/components/Jobs/Jobs.scss

```diff
@@ -239,3 +239,12 @@
     }
   }
 }
+
+.no-results-message {
+  flex-grow: 1;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding-top: 200px;
```

Comment:
I wonder if we can find another solution besides `padding-top`. For windows with a shorter height, a hardcoded value may make the message invisible.

Reply:
I removed the static padding value for now and added a `Section` component which creates some padding at the top so that the message does not look awkward. I realize that the ideal way to do this would require some tweaking in the `InfiniteScrollTable` component so that it can properly position elements that come in with the `noDataText` prop. This will likely have an effect in other places where InfiniteScrollTable is used as well so this task would be out of scope for this PR. I can create a task to address this in the future.